mantidproject/mantid
# Add WAND²

Add the new WAND² instrument. mantidgeometry was updated mantidproject/mantidgeometry@5611b495ab5f18b3fd9220247f632d5295f398ba

**To test:**
I suggest loading at a data file from `/HFIR/HB2C/IPTS-7776/nexus/` with the IDF in the correct place as it requires logfiles to correctly appear.

You should also be able to connect to the live data but it will fail as it's currently sending an invalid IDF.

**Release Notes** 
*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
